.changeset/small-doors-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-internal/scripts': minor
```

Comment:
Do we care to have changesets for private/internal packages? I can see it potentially having value for tracking changes on this without looking at the git history. It won't be published either way.

Reply:
Great catch! I blindly followed Changeset but 100% agree that in the release we prefer to not have this.